dbus/org.nemo.transferengine.xml

```diff
@@ -45,6 +45,7 @@
             <arg direction="in" type="as" name="callback"/>
             <arg direction="in" type="s" name="cancelMethod"/>
             <arg direction="in" type="s" name="restartMethod"/>
+            <arg direction="in" type="b" name="privateTransfer"/>
```

Comment:
I'm a bit vary of breaking the D-Bus and c++ api here. One alternative could be adding separate methods for the private thing.

Referring the comment I made about whether this should be in the database: if this doesn't end up in the database, it could be maybe called 'transient' instead of 'private. There could be reasons to use it for transfers which are not per se private but would be better left out of transfer log anyway.